### DISCUSSION TITLE
Weather Forecast

### DISCUSSION THREAD

**Discussion by Greg-ATG** [bluecaret/carettab#361] [Help and Support]
What is the source for the weather forecast?  We have our correct location set and the CURRENT weather is accurate.  However the forecast temperatures are usually wildly different than any other weather source predicts.  E.g. this morning, the forecasted high for Friday was 10 degrees higher than weather.com, although right now it's only showing 5 degrees higher (not sure why the forecast would change so dramatically within a few hours).  Either the source is inaccurate or perhaps there's a bug with taking in and presenting the values?

**Comment by bluecaret** [comment]
I use weatherapi.com which I agree has not been accurate lately, I've reached out them and am waiting a reply right now. Unfortunately not much I can do about it other than that. I would love to use a more well known source but the pricing quickly adds up for the data.

**Reply by Greg-ATG** [reply]
Understood, thanks for the transparency.

**Reply by bluecaret** [reply]
Can you let me know if you are seeing improved data in the weather widget now? The API is saying things have been fixed. I'm seeing better data in my location now. Thank you.

**Reply by Greg-ATG** [reply]
The forecasts look a little better, but still seem to be 2-5 degrees F different from a "trustworthy" forecast.  We're just going to stick with it showing the current weather for now.

**Reply by bluecaret** [reply]
Can I ask what source you are comparing to? I know that different sources can vary around that amount as it depends on what specific location they are pulling from. I did look into some different API to use, and only one of them was in an affordable range for me, unfortunately when I tested a few areas I always use for weather it did not have those locations so I would be afraid of it having less options for users. Right now I'm not sure I'll be changing services after all as my options are very limited.

Thank you for all the info, it does help! And sorry for the lackluster weather data.

**Comment by Greg-ATG** [comment]
I was comparing to weather.com previously.  Comparing to a few different sources (NWS, Weather Underground, and AccuWeather) right now for where we are (Eastern US) actually shows that it's pretty good--only 1 degree away from any of these other sources.  We may try it out again.

**Reply by bluecaret** [reply]
Thank you very much for the extra info, it is appreciated. Also glad it is working better now.
